documentation/Using-the-SDK/Building-a-table.md

```diff
@@ -1,21 +1,27 @@
 # Building a Table
 
-Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to add both Columns and `TableConfigurations`.
+Table construction happens by interacting with an instance of an `ITableBuilder`. Broadly, `Tables` consist of `TableConfigurations` and Columns. Columns can be further broken down into `ColumnConfigurations` and `Projections`. An `ITableBuilder` has methods to add `TableConfigurations` and to establish a row count for the table.
 
-* [Column](#column)
-  * [ColumnConfiguration](#columnconfiguration)
-  * [Projection](#projection)
-  * [Combining ColumnConfiguration and Projections](#combining-columnconfiguration-and-projections)
-* [TableConfiguration](#tableconfiguration)
-  * [ColumnRole](#columnrole)
+*Note*: Every projection in the table must support the table's established row count.
```

Comment:
I think it might make sense to move this note in the "Adding columns" section.

Also, we should make it more in-your-face:

> ❗ Every projection in the table must support the table's established row count

Reply:
sure

Reply:
wait, what Adding columns section?